infra/experimental/sanitizers/ExecSan/execSan.cpp

```diff
@@ -66,6 +66,12 @@ const std::string kTripWire = "/tmp/tripwire";
 const std::string kInjectionError = "Shell injection";
 // Shell corruption bug detected based on syntax error.
 const std::string kCorruptionError = "Shell corruption";
+// The magic string that we'll use to detect arbitrary file open
+const std::string kFzAbsoluteDirectory = "/fz/";
+// Arbitrary file open in /fz/
+const std::string kArbitraryFileOpenError = "Arbitrary file open";
+// Assuming we will a shorter top dir.
```

Comment:
This comment isn't complete. I'm not sure what it is trying to say.

Reply:
improved it indeed.